modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -165,28 +169,36 @@ object StepsTable {
 
     // Columns for the table
     private def columns(p: Props, s: Size): List[Table.ColumnArg] = {
-      val (offsetVisible, exposureVisible, fpuVisible, filterVisible, objectSize) = s.width match {
-        case w if w < PhoneCut      => (false, false, false, false, SSize.Tiny)
-        case w if w < LargePhoneCut => (false, true, false, false, SSize.Small)
-        case _                      => (displayOffsets(p), true, true, true, SSize.Small)
+      val (offsetVisible, exposureVisible, disperserVisible, fpuVisible, filterVisible, objectSize) = s.width match {
+        case w if w < PhoneCut      => (false, false, false, false, false, SSize.Tiny)
+        case w if w < LargePhoneCut => (false, true, false, false, false, SSize.Small)
+        case _                      => (displayOffsets(p), true, true, true, true, SSize.Small)
       }
       val offsetColumn =
         p.offsetsDisplay match {
           case OffsetsDisplay.DisplayOffsets(x) =>
             Column(Column.props(ColWidths.OffsetWidthBase + x, "offset", label = "Offset", flexShrink = 0, disableSort = true, cellRenderer = stepStatusRenderer(p.offsetsDisplay))).some
           case _ => None
         }
-        List(
+
+        val hasDisperser = p.steps.exists(s => s.instrument === Instrument.GmosS || s.instrument === Instrument.GmosN)
```

Comment:
Yeah this isn't very nice. I thin we should make this a property of the instrument, on the `Model` class and do the filtering based on that

Reply:
Yes, this definitely is not the way to do it. I didn't want to make any substantial changes until I confirmed it was working with this easy hack, and until I talked to you to see what you thought was the best way moving forward.

When you say "do the filtering based on that", what do you mean?  I'm not sure about this list of columns. Will they be substantially different as we continue to add instruments?

Reply:
Maybe the correct word isn't `filtering`. I think instruments can have a `hasDisperser` property and we include the disperser column based on that

Reply:
Sure thing! Will do.